pyclient/stix_shifter_dataframe.py

```diff
@@ -56,7 +56,8 @@ def stix_shiter_execute(self, config_name: str, stix_query: str):
                         logging.debug(status)
                     else:
                         raise RuntimeError("Fetching status failed")
-                result = transmission.results(search_id, 0, 9)
+                # TODO: allow user to pass No. of records
```

Comment:
Can you create an issue for this TODO?

Reply:
https://github.com/IBM/ibm-security-notebooks/issues/12